mintlify/api-reference/environments.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Environments'
+---
+
+Lightspark Grid provides two environments: production and sandbox.  Both environments use the same base URL and API tokens are scoped to an environment.
```

Comment:
Should we keep all references as UMAaaS for now until we switch naming globally?

Reply:
I was thinking of updating it to a variable like {topLevelProductName}.  But didn't spend enough time looking at variables in Mintlify.